dragn/tests/test_dice.py

```diff
@@ -34,6 +34,16 @@ def test_all_configured_dice() -> None:
             assert die.max_value == max(list(results_range))
             assert str(die) == f"D{die.max_value}"
 
+    @staticmethod
+    def test_zero_sided_dice_fails() -> None:
```

Comment:
Looks great!! Just one tiny thing, the test bellow, you named it the scenario, I love that, it tells people reading the tests what it is doing, and when they read the test, they can see clearly that it raises an attribute error.

But in this one, the name contains the expected result. To keep things consistent with all other tests, could you just remove the `fails` bit in the end of the function?